README.md

```diff
@@ -21,6 +21,15 @@ The XLT build process is based on the **Apache Ant** build tool. The build works
         * [chromedriver](https://chromedriver.chromium.org/)
         * [geckodriver](https://github.com/mozilla/geckodriver)
 
+## IDE Import
+* Import the project as `Maven` project into your IDE.
+* After importing a module package conflict might be blamed. In that case go to your IDE's build path settings and remove package `jdk.sobject` from module dependencies.
```

Comment:
IMO there is no need for using an unordered list here as those items are (nearly) paragraphs anyway. 
```suggestion
Follow your IDE's instructions on how to import an existing Maven project.

Once import of the project has completed, a module package conflict might be blamed. In that case, go to your IDE's build path settings and remove module `jdk.jsobject` from the module dependencies.
```

Also, please add a blank line between headline and text.